src/main/resources/interface-specification.yml

```diff
@@ -256,6 +256,95 @@ paths:
         description: payload for agreement creation
         required: true
       description: creates the agreement between the involved parties.
+  '/agreements/filter/producers':
```

Comment:
[same](https://github.com/pagopa/interop-be-agreement-process/pull/217#discussion_r1104129883)

Reply:
unfortunately here we cannot use the path `/producers` because it is already taken.
I'm yet again invoking your wisdom @beetlecrunch @TonioGela

Reply:
Why not convey this to `/producers `request, using different query params?
For example
- /producers?from=agreements
- /producers?from=eservices
Parameter from is mandatory and accepts only a single value.

Reply:
I'm not sure, it doesn't seem very RESTful, and I don't know if we can guarantee that the response payload will always be the same.
That said, I do not have an alternative, so as of now yours is the best one 🤣

Reply:
How about? 
```
/producers?eservices=yada,yada
/producers?agreements=yada,yada
```

Reply:
In this case there is ambiguity, what if both `eservices` and `agreements` are passed?